[2023-07-08T11:11:41.751+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: assignment.get_departed_flights_from_cdg manual__2023-07-08T11:11:40.461764+00:00 [queued]>
[2023-07-08T11:11:41.769+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: assignment.get_departed_flights_from_cdg manual__2023-07-08T11:11:40.461764+00:00 [queued]>
[2023-07-08T11:11:41.770+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-08T11:11:41.805+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonDecoratedOperator): get_departed_flights_from_cdg> on 2023-07-08 11:11:40.461764+00:00
[2023-07-08T11:11:41.815+0000] {standard_task_runner.py:57} INFO - Started process 603 to run task
[2023-07-08T11:11:41.821+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'assignment', 'get_departed_flights_from_cdg', 'manual__2023-07-08T11:11:40.461764+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/assignment.py', '--cfg-path', '/tmp/tmpuc12m4fy']
[2023-07-08T11:11:41.822+0000] {standard_task_runner.py:85} INFO - Job 60: Subtask get_departed_flights_from_cdg
[2023-07-08T11:11:41.895+0000] {task_command.py:410} INFO - Running <TaskInstance: assignment.get_departed_flights_from_cdg manual__2023-07-08T11:11:40.461764+00:00 [running]> on host da250f106141
[2023-07-08T11:11:42.011+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='assignment' AIRFLOW_CTX_TASK_ID='get_departed_flights_from_cdg' AIRFLOW_CTX_EXECUTION_DATE='2023-07-08T11:11:40.461764+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-08T11:11:40.461764+00:00'
[2023-07-08T11:11:42.422+0000] {logging_mixin.py:149} INFO - heeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeerrrrrrrrrrrrrrrrrrrrrrrrreeeeeeeeeeeeeeeeeeeeeeeeeeeee
[2023-07-08T11:11:42.424+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 220, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/assignment.py", line 76, in get_departed_flights_from_cdg
    'departures_list': json.dump(departures),
TypeError: dump() missing 1 required positional argument: 'fp'
[2023-07-08T11:11:42.435+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=assignment, task_id=get_departed_flights_from_cdg, execution_date=20230708T111140, start_date=20230708T111141, end_date=20230708T111142
[2023-07-08T11:11:42.449+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 60 for task get_departed_flights_from_cdg (dump() missing 1 required positional argument: 'fp'; 603)
[2023-07-08T11:11:42.473+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-07-08T11:11:42.514+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
